tests/conftest.py

```diff
@@ -1,11 +1,11 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
+#!/usr/bin/env python
 
-import pytest
-import sys
 import os
-sys.path.append(os.path.abspath(os.path.dirname(os.path.abspath(__file__)) + '/../src/'))
+import sys
 
+import pytest
+
+sys.path.append(os.path.abspath(os.path.dirname(os.path.abspath(__file__)) + '/../src/'))
```

Comment:
Python3.4以降、os.pathよりpathlibを使うことが推奨されている。
cf. https://qiita.com/tag1216/items/29bf3b9c3c4b563ff0fe

```suggestion
import pathlib
src_dir= pathlib.Path(__file__) / '../src'
sys.path.append(src_dir.resolve())
```

pathlibの方がsimpleに書ける。